tests/test_dataset.py

```diff
@@ -129,6 +138,7 @@ def test_dataset_save_to_disk(custom_dataset):
         assert (ds_name / "task.pkl").is_file()
```

Comment:
Could you update the "task.pkl" to the proper variable name? That's my bad... I'm updating this in a different PR, and the change will ensure this test still works as expected: 
```python
from distilabel.utils.dataset import TASK_FILE_NAME
```